sdk/Lusid.Sdk.Tests/Utilities/TestDataUtilities.cs

```diff
@@ -19,6 +19,7 @@ public static class TestDataUtilities
         public static string ValuationPvKey = "Valuation/PV/Amount";
         public static string InstrumentName = "Instrument/default/Name";
         public static readonly string ValuationPv = "Valuation/PV/Amount";
+        public static readonly string ValuationPvInReportCcy = "Valuation/PvInReportCcy";
```

Comment:
Necessary to get portfolio PV when there's multiple currencies of cash in the portfolio.